api/src/main/java/org/openmrs/module/appointments/service/impl/AppointmentsServiceImpl.java

```diff
@@ -21,11 +23,18 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.Set;
 import java.util.stream.Collectors;
 
+import static java.util.Objects.isNull;
+
+import static org.openmrs.module.appointments.constants.PrivilegeConstants.MANAGE_APPOINTMENTS;
+import static org.openmrs.module.appointments.model.AppointmentProviderResponse.ACCEPTED;
```

Comment:
I don't think this ever used.

Reply:
Agreed. We will remove it.

Reply:
No you don't have to mention them twice. I probably missed them in my first review. I'll have a look again.